hoodie-common/src/main/java/com/uber/hoodie/common/util/FSUtils.java

```diff
@@ -109,7 +110,12 @@ public static String getCommitFromCommitFile(String commitFileName) {
     }
 
     public static String getCommitTime(String fullFileName) {
-        return fullFileName.split("_")[2].split("\\.")[0];
+        Matcher matcher = LOG_FILE_PATTERN.matcher(fullFileName);
+        if (!matcher.find()) {
+            return fullFileName.split("_")[2].split("\\.")[0];
+        } else {
+            return fullFileName.split("_")[1].split("\\.")[0];
```

Comment:
I looked into using FileGroups briefly but then realized this fix needs to be backported to the hoodie-mor-mvp release which does not have the concept of FileGroups. It will need to be fixed in someway without the filegroups first and then I can send another PR to use filegroups.